### DISCUSSION TITLE
TransitionHelper - An animation helper that morphs between two controls

### DISCUSSION THREAD

**Discussion by HHChaos** [CommunityToolkit/Labs-Windows#353] [Experiments]
## Describe the problem this feature would solve

Hi, I am a member of the Microsoft store team. we created a cool helper in the store for transitioning from one control to another. For example, we scroll down on a product page, and we can see the header of this page transformed to a sticky bar fixed at the top. This helper helps us to transform the image, title, buttons and other controls in the header to the corresponding positions in the sticky bar. When we back to the top of the page, it also has a good reverse animation.

![Animation](https://user-images.githubusercontent.com/9880072/137608730-145cb74a-dab5-485b-84f6-995e13078a96.gif)

The code is built on top of Sergio’s awesome `AnimationBuilder`, and we would love to contribute back to the community by opening source it. What do you think of this?

**Comment by michael-hawker** [comment]
Initial PR to Labs here: https://github.com/CommunityToolkit/Labs-Windows/pull/354
